docs/web-ui.md

```diff
@@ -407,6 +407,20 @@ Here is the list of SQL metrics:
 
 </table>
 
+## Structured Streaming Tab
+When running Structured Streaming jobs in micro-batch mode, a Structured Streaming tab will be
+available on the Web UI. You can check the statistics of a running query or a completed query 
+via clicking a run id on the Streaming Query overview page.
```

Comment:
We can also check the latest exception for the failed query.

Reply:
Oh, sorry for missing this part. I'll add this.